src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -631,7 +632,7 @@ static inline int MPIDI_NM_mpi_win_lock(int lock_type, int rank, int assert, MPI
 #define FUNCNAME MPIDI_NM_mpi_win_unlock
 #undef FCNAME
 #define FCNAME MPL_QUOTE(FUNCNAME)
-static inline int MPIDI_NM_mpi_win_unlock(int rank, MPIR_Win * win, MPIDI_av_entry_t *addr)
+static inline int MPIDI_NM_mpi_win_unlock(int rank, MPIR_Win * win, MPIDI_av_entry_t * addr)
```

Comment:
I guess this section does not specify the whitespace format for variable declaration.  In fact, running `./maint/code-cleanup.sh` script is always recommended (see [here](https://wiki.mpich.org/mpich/index.php/Coding_Standards#Code_Cleanup_Script)) for code format consistency.